docs/deploy/README.md

```diff
@@ -0,0 +1,87 @@
+# FinOps toolkit deployment options
+
+![Status: Not started](https://img.shields.io/badge/status-not%20started-red) &nbsp;<sup>→</sup>&nbsp;
+[![#1](https://img.shields.io/github/issues/detail/state/microsoft/cloud-hubs/1)](https://github.com/microsoft/cloud-hubs/issues/1)
+
+The FinOps toolkit includes multiple ARM templates depending on your needs. We recommend using the **FinOps hub with exports** template to streamline your setup process. All the examples below refer to this template. You may select a different template, which will change the available options.
+
+For more details about toolkit options, refer to the [available templates](../templates).
+
+> ### ⚠️ Important
+>
+> _The FinOps toolkit will change over time. We highly recommend saving a parameter file to ensure you can re-deploy new versions of the template with the same parameters._
+
+On this page:
+
+- [Using custom deployment in the Azure portal](#using-custom-deployment-in-the-azure-portal)
+- [Deploy from Microsoft Learn code samples](#deploy-from-microsoft-learn-code-samples)
+- [Deploy from Azure Quickstart Templates](#deploy-from-azure-quickstart-templates)
+- [Future considerations](#future-considerations)
+
+---
+
+## Using custom deployment in the Azure portal
+
+The Azure portal includes a **Custom deployment** option that supports all templates available in the Azure Quickstart Templates repository. To deploy a quickstart template:
+
+1. Open [Custom deployment](https://portal.azure.com/#create/Microsoft.Template)
```

Comment:
Not sure if it's important but the option in portal is "Deploy a custom template" than on the actual page as you point out it's "Custom deployment" might confuse some users the difference.

Reply:
I can share the feedback with the portal team. I'm just documenting what they have right now. I plan to add a separate proposal for what we should do in CM where we can do more.